clang/include/clang/CIR/Interfaces/CMakeLists.txt

```diff
@@ -20,6 +20,15 @@ function(add_clang_mlir_op_interface interface)
   add_dependencies(mlir-generic-headers MLIR${interface}IncGen)
 endfunction()
 
+function(add_clang_mlir_type_interface interface)
+  set(LLVM_TARGET_DEFINITIONS ${interface}.td)
+  mlir_tablegen(${interface}.h.inc -gen-type-interface-decls)
+  mlir_tablegen(${interface}.cpp.inc -gen-type-interface-defs)
+  add_public_tablegen_target(MLIRCIR${interface}IncGen)
+  add_dependencies(mlir-generic-headers MLIRCIR${interface}IncGen)
+endfunction()
+
 add_clang_mlir_attr_interface(ASTAttrInterfaces)
 add_clang_mlir_op_interface(CIROpInterfaces)
 add_clang_mlir_op_interface(CIRLoopOpInterface)
+add_clang_mlir_type_interface(FPTypeInterface)
```

Comment:
Please use the convention used for LoopOp, should start with CIR

Reply:
Updated.